config/autoload/.gitignore

```diff
@@ -1,2 +1,3 @@
 local.php
 *.local.php
+event_store.local.php.copy
```

Comment:
Don't we normally call these ".dist" https://github.com/prooph/proophessor-do/tree/master/config/autoload